client/question/services/code_evaluators/PythonPrereqCheckService.js

```diff
@@ -31,23 +31,36 @@ tie.factory('PythonPrereqCheckService', [
       WRONG_LANGUAGE_ERRORS, PREREQ_CHECK_TYPE_INVALID_SYSTEM_CALL,
       PREREQ_CHECK_TYPE_INVALID_AUXILIARYCODE_CALL, CLASS_NAME_AUXILIARY_CODE,
       CLASS_NAME_SYSTEM_CODE) {
+
+    /**
+     * Returns a list of lines in the code that do not contain strings, in
+     * order to prevent spurious false positives in code-analysis checks.
+     *
+     * NOTE TO DEVELOPERS: This check assumes that the code does not contain
```

Comment:
Should we add a TODO to fix that in the future? This seems like it could cause some issues down the line otherwise.

Reply:
Good idea; done.